### PR TITLE
Add eval details template for mixed scripts

### PR DESCRIPTION
**Pull request by eleftherias**
# Summary
Add a template for the evaluation details message of `homoglyphs/mixed_scripts`

Fixes https://github.com/stacklok/minder-stories/issues/39

## Change Type

***Mark the type of change your PR introduces:***

- [ ] Bug fix (resolves an issue without affecting existing features)
- [x] Feature (adds new functionality without breaking changes)
- [ ] Breaking change (may impact existing functionalities or require documentation updates)
- [ ] Documentation (updates or additions to documentation)
- [ ] Refactoring or test improvements (no bug fixes or new functionality)

# Testing

***Outline how the changes were tested, including steps to reproduce and any relevant configurations. 
Attach screenshots if helpful.***

# Review Checklist:

- [x] Reviewed my own code for quality and clarity.
- [x] Added comments to complex or tricky code sections.
- [ ] Updated any affected documentation.
- [x] Included tests that validate the fix or feature.
- [ ] Checked that related changes are merged.

### CODE REVIEW

File: internal/engine/eval/homoglyphs/application/mixed_scripts_eval_test.go [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+							ScriptsFound: []string{"Cyrillic", "Latin"},
+						},
+					},
+					{
+						MixedScript: &domain.MixedScriptInfo{
+							Text:         "Ѳ.HandleFunc(\"hi\",",
+							ScriptsFound: []string{"Cyrillic", "Latin"},
+						},
+					},
+				},
```
Comment (blkt) [inline internal/engine/eval/homoglyphs/application/mixed_scripts_eval_test.go:75]: **nit:** what do you think about adding a test having `Text: ""` and another one having `ScriptsFound: nil` as a sort of documentation?

**Review by blkt — APPROVED** [review]
Minor comment, overall LGTM!

File: internal/engine/eval/homoglyphs/application/mixed_scripts_eval_test.go (hunk 1 above)
Comment (eleftherias) [reply internal/engine/eval/homoglyphs/application/mixed_scripts_eval_test.go:75]: I would guess that we would never be in that state because if the text or scripts aren't present, then we shouldn't have a mixed scripts violation.

Comment (eleftherias) [reply internal/engine/eval/homoglyphs/application/mixed_scripts_eval_test.go:75]: I'm going to merge this so that I can build on top of it for the other homoglyphs rule but I'm happy to continue the discussion here.
